src/sentry/integrations/base.py

```diff
@@ -178,6 +183,27 @@ def get_client(self):
         # Return the api client for a given provider
         raise NotImplementedError
 
+    def get_default_identity(self):
+        """
+        For Integrations that rely solely on user auth for authentication
+        """
+        if self.organization_id is None:
+            raise NotImplementedError
+
+        try:
+            org_integration = OrganizationIntegration.objects.get(
+                organization_id=self.organization_id,
+                integration_id=self.model.id,
+            )
+        except OrganizationIntegration.DoesNotExist as e:
+            raise e
```

Comment:
i don't think there's a point of having these `except` blocks if you're just going to re-raise, is there?

Reply:
No you're right. I just wasn't sure what else to write there. ^ . ^' I can remove them.